src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,50 @@
+package controller;
+
+import static util.InputRetryHelper.inputRetryHelper;
+import static view.OutputView.printChessBoard;
+import static view.OutputView.printInitialGamePrompt;
+
+import java.util.List;
+import model.chessboard.ChessBoard;
+import model.position.Position;
+import view.GameCommand;
+import view.InputView;
+import view.dto.InfoMapper;
+import view.dto.PieceInfo;
+
+public class Controller {
+
+    public void execute() {
+        GameCommand gameCommand = executeInitial();
+        ChessBoard chessBoard = new ChessBoard();
+        while (gameCommand != GameCommand.END) {
+            List<PieceInfo> pieceInfos = InfoMapper.toPieceInfoMapper(chessBoard);
+            printChessBoard(pieceInfos);
+            gameCommand = inputRetryHelper(() -> runGame(chessBoard));
+        }
+    }
+
+    private GameCommand runGame(ChessBoard chessBoard) {
+        List<String> inputCommand = InputView.parseCommand();
+        GameCommand gameCommand = inputRetryHelper(() -> GameCommand.from(inputCommand));
+        if (gameCommand == GameCommand.MOVE) {
+            controlChessBoard(chessBoard, inputCommand);
+        }
+        return gameCommand;
+    }
+
+    private void controlChessBoard(ChessBoard chessBoard, List<String> inputCommand) {
+        int file = GameCommand.toSourceFileValue(inputCommand);
+        int rank = GameCommand.toSourceRankValue(inputCommand);
+        Position source = Position.of(file, rank);
+        int toFile = GameCommand.toDestinationFileValue(inputCommand);
+        int toRank = GameCommand.toDestinationRankValue(inputCommand);
+        Position destination = Position.of(toFile, toRank);
```

Comment:
```suggestion
        Position source = GameCommand.toPosition(inputCommand.get(1));
        Position destination = GameCommand.toPosition(inputCommand.get(2));
```

가능하면 공통으로 사용할 수 있는 범용적인 메서드를 만들어 봅시다. 메서드를 4개에서 1개로 줄일 수 있습니다.

Reply:
반영하지 않은 이유가 있을까요?

Reply:
1. 이동 Commad인 `move src dest `는 그 입력 순서(format)가 명확한 View 단의 입력 요구사항이라고 생각했습니다!

따라서 Controller가 Command에 존재하는 index를 통해 get을 사용하여 파라미터로 넘겨주면 view의 요구사항을 알고 침범하기에 명확한 역할 분리를 위해 코드를 분리했습니다.

2. GameCommand 클래스는 입력된 Command를 처리 및 변환하여 Controller에게 그 정보를 넘겨주는 역할을 가지고 있습니다.
그래서 도메인에서 가장 주요 로직으로 사용되는 `Position` 객체를  `GameCommand`가 직접 생성하지 않도록 분리가 필요하다고 생각했습니다.

위의 각 두 이유에 의해 Source와 Destination을 각각 구하는 메서드 2개가 필요했고, 각 File과 Rank를 구하는 메서드 2개가 필요해서 4개의 메서드로 나눴습니다!